MSBuild.Dev.sln

```diff
@@ -3,6 +3,12 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27004.2009
 MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{4900B3B8-4310-4D5B-B1F7-2FDF9199765F}"
```

Comment:
This snippet is in MSBuild.sln but not in MSBuild.dev.sln. So I copied it over. No idea what it does, but why not? :)

Reply:
It makes these things appear in the sidebar:

![image](https://user-images.githubusercontent.com/3347530/80039484-88618800-84bd-11ea-9524-f6631e40d88c.png)

Reply:
That's the first degree effect, but does it lead to those files getting picked up by anything else in VS? Forever a mystery.

Reply:
I don't think it should but I'm not willing to say it with that much confidence.